modules/extensions/scripts/install_OPA_gatekeeper.template.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2017, 2021 Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+if [ ! -f .OPA_completed ]; then
+
+  echo "Installing OPA Gatekeeper"
```

Comment:
I recommend using the full product name, i.e. "Open Policy Agent" instead of the acronym, unless it's already been used in a previous output.

Reply:
Done.